e2e_samples/dataset_versioning/infra/README.md

```diff
@@ -0,0 +1,138 @@
+# Infrastructure as Code
+
+## Getting Started
+
+**Prerequisites:**
+
+- Terraform > 0.14.7
+- Azure cli > 2.19.1
+
+**Deploy Steps:**
+
+The following deploys the Azure base infrastructure for the Temperature Events sample. Ensure you are in `e2e_samples/dataset-versioning/infra/`.
+
+1. `az login`
```

Comment:
This summary is a bit confusing.
3 of them are actual commands and one is a description.

It might make more sense to for these all to just describe what the step is, the commands are below anyway.